collector/ntp.go

```diff
@@ -26,7 +26,8 @@ import (
 )
 
 var (
-	ntpServer = flag.String("collector.ntp.server", "", "NTP server to use for ntp collector.")
+	ntpServer          = flag.String("collector.ntp.server", "", "NTP server to use for ntp collector.")
+	ntpProtocolVersion = flag.Int("collector.ntp.protocol", 4, "NTP protocol version")
```

Comment:
Thanks!

How about calling this `collector.ntp.protocol-version` (or just `collector.ntp.version`?) to make the flag name more self-explanatory?